rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java

```diff
@@ -98,20 +104,35 @@ public static void main(String[] args) throws Exception {
 
     public static class EnrichmentFunction
             extends RichCoFlatMapFunction<TaxiRide, TaxiFare, RideAndFare> {
+        private ValueState<TaxiRide> taxiRide;
+        private ValueState<TaxiFare> taxiFare;
```

Comment:
All Flink state must be made transient so that the state value is not serialized and not shared across multiple Flink task managers. https://stackoverflow.com/a/72558397

Reply:
Thanks, will update to make state transient.